src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeChunkRowState.java

```diff
@@ -136,7 +136,15 @@ protected void removeClazz()
    @Override
    protected void setTitle(String title)
```

Comment:
This change feels a little strange. Should we instead have a function like `appendToTitle()` for the cases where we want to append content?

Reply:
Or instead, the caller of `setTitle()` should be querying the current title and appending to it instead; I don't think that should be happening here.

Reply:
I like the suggestion of adding `appendToTitle()`